packages/core/src/index.ts

```diff
@@ -279,7 +278,9 @@ class Hopr extends EventEmitter {
     this.status = 'INITIALIZING'
     log('Starting hopr node...')
 
-    const balance = await this.connector.getNativeBalance(false)
+    const connector = HoprCoreEthereum.instance
```

Comment:
Can the singleton instance be accessed with `.instance` or only with `getInstance()`?

Reply:
I need to change this to `getInstance()` only, because of the Rust interop issues.